packages/gestalt/src/Upsell.js

```diff
@@ -99,7 +99,12 @@ export default function Upsell({
           mdMarginBottom={primaryLink || secondaryLink ? 0 : undefined}
           mdPaddingY={3}
         >
-          <Box marginBottom="auto" marginTop="auto" maxWidth={648} paddingX={6}>
+          <Box
+            marginBottom="auto"
+            marginTop="auto"
+            maxWidth={648}
+            marginEnd={6}
```

Comment:
For reference, 6 boints  because that was what we had on Callout, with a paddingX=6 around the message
![image](https://user-images.githubusercontent.com/10593890/98309390-23fca680-1f7f-11eb-97d7-97890e7c2b52.png)